src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,20 +12,39 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
+    public static final String EXAMPLE = "123, #12-645, some street, 213551";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses should have Block, Unit, Street and Postal Code";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
 
     public final String value;
     private boolean isPrivate;
+    private Block block;
+    private Unit unit;
+    private Street street;
+    private PostalCode postalCode;
+    private final addressLimit = 4;
```

Comment:
if you intend to make this like a "constant", the variable name must be in all caps according to the coding standards we follow